packages/router/jest.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  setupFiles: ['./src/__tests__/global.setup.js'],
+}
```

Comment:
I think `global.scrollTo = jest.fn()` can be handled directly in this file.

Reply:
Ah, sorry, turns out I'm off base here. This file is probably needed, but maybe you can just place it in the root and call it `jest.setup.js`